draft-ietf-quic-multipath.md

```diff
@@ -1091,6 +1109,57 @@ field. If the Identifier Type if 0x02, the PATH_ABANDON frame MUST only
 be sent on the path that is intended to be abandoned.
 
 
+## PATH_STATUS frame {#path-status-frame}
+
+PATH_STATUS Frame are used by endpoints to inform the peer of the current
+status of one path, and the peer should send packets according to
+the preference expressed in these frames.
+PATH_STATUS frames are formatted as shown in {{fig-path-status-format}}.
+
+~~~
+  PATH_STATUS Frame {
+    Type (i) = TBD-03 (experiments use 0xbaba06),
+    Path Identifier (..),
+    Path Status sequence number (i),
+    Path Status (i),
+  }
+~~~
+{: #fig-path-status-format title="PATH_STATUS Frame Format"}
+
+PATH_STATUS Frames contain the following fields:
+
+Path Identifier: An identifier of the path, which is formatted
+  as shown in {{fig-path-identifier-format}}. Exactly the same as
+  the definition of Path Identifier in {#path-abandon-frame}.
+
+Path Status sequence number: A variable-length integer specifying
+  the sequence number assigned for this PATH_STATUS frame. The sequence
+  number MUST be monotonically increasing for the sender of
+  the Path Status frame in the same connection.
```

Comment:
Maybe we should state that the scope of the sequence number is relative to the Path Identifier? Having a single space for the whole connection might cause issues if we have different paths with different latencies.

Reply:
Actually later this is explained, but I think it could be useful to explicitly state that here.

Reply:
Actually the receiver should maintain the largest received sequence number for each path. The receiver should accept the Path Status Frame if and only if its sequence number is larger than the largest received sequence number of the Path Identifier. The latencies of different paths doesn't matter, because the receiver can distinguish Status Frames for different paths with Path Identifier field, and compares the sequence numbers independently.

Reply:
I fully agree with your description, I just suggested to make it clearer here.
```suggestion
  the sequence number assigned for this PATH_STATUS frame. The sequence
  number is relative to the Path Identifier and MUST be monotonically increasing
  for the sender of the Path Status frame in the same connection.
```

Reply:
I updated the description to make it clearer, and it's different for the sender and receiver side.